test/e2e/test-apps/sessions/native-crash-renderer/session-next.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I'm not 100% sure why this extra sessionm update gets sent.

It's another `ok` status with an `id` that matches the current session but with `init: false`.

Reply:
Is this from calling `checkPreviousSession` maybe?